StripeFinancialConnections/StripeFinancialConnections/Source/Native/PartnerAuth/PartnerAuthViewController.swift

```diff
@@ -261,86 +390,12 @@ final class PartnerAuthViewController: UIViewController {
                     let urlComponsents = URLComponents(url: returnUrl, resolvingAgainstBaseURL: true),
                     let status = urlComponsents.queryItems?.first(where: { $0.name == "status" })?.value
                 {
-                    if status == "success" {
-                        self.dataSource.recordAuthSessionEvent(
-                            eventName: "success",
-                            authSessionId: authSession.id
-                        )
-                        
-                        if authSession.isOauthNonOptional {
-                            // for OAuth flows, we need to fetch OAuth results
-                            self.authorizeAuthSession(authSession)
-                        } else {
-                            // for legacy flows (non-OAuth), we do not need to fetch OAuth results, or call authorize
-                            self.delegate?.partnerAuthViewController(self, didCompleteWithAuthSession: authSession)
-                        }
-                    } else if status == "failure" {
-                        self.dataSource.recordAuthSessionEvent(
-                            eventName: "failure",
-                            authSessionId: authSession.id
-                        )
-                        
-                        // cancel current auth session
-                        self.dataSource.cancelPendingAuthSessionIfNeeded()
-                        
-                        // show a terminal error
-                        self.showErrorView(
-                            FinancialConnectionsSheetError.unknown(
-                                debugDescription: "Shim returned a failure."
-                            )
-                        )
-                    } else { // assume `status == cancel`
-                        self.dataSource.recordAuthSessionEvent(
-                            eventName: "cancel",
-                            authSessionId: authSession.id
-                        )
-                        
-                        // cancel current auth session
-                        self.dataSource.cancelPendingAuthSessionIfNeeded()
-                        
-                        // whether legacy or OAuth, we always go back
-                        // if we got an explicit cancel from backend
-                        self.navigateBack()
-                    }
+                    self.handleAuthSessionCompletion(status, authSession)
```

Comment:
to match `handleAuthSessionCompletionWithNoStatus` naming

it may be a better name to call `handleAuthSessionCompletion` -> `handleAuthSessionCompletionWithStatus`

this is mostly stylistic as it creates a "family of naming," but if one is extremely logical, `handleAuthSessionCompletion` is a bit misleading when `handleAuthSessionCompletionWithNoStatus` exists because it only handles auth session when there is a status. (though that is enforced via parameters/compiler)